connector/fyle_integrations_platform_connector/apis/merchants.py

```diff
@@ -0,0 +1,85 @@
+from heapq import merge
+from traceback import print_tb
+
+from apps.workspaces.models import Workspace
+from .base import Base
+from typing import List
+from fyle_accounting_mappings.models import ExpenseAttribute
+
+class Merchants(Base):
+    """
+    Class for Merchants API
+    """
+    def __init__(self):
+        Base.__init__(self, attribute_type='MERCHANT', query_params={'field_name':'eq.Merchant'})
+    
+    def construct_query_params(self) -> dict:
+        """
+        Constructs the query params for the API call.
+        :return: dict
+        """
+        params = {'order': 'updated_at.desc'}
+        params.update(self.query_params)
+
+        return params
+
+    def get(self):
+        generator = self.get_all_generator()
+        for items in generator:
+            merchant = items['data'][0]
+        
+        return merchant
+
+    def post(self, payload: List[str]):
+        """
+        Post data to Fyle 
+        """
+        generator = self.get_all_generator()
+        for items in generator:
+            merchant = items['data'][0]
+            merchant['options'].extend(payload)
+            merchant_payload = { 
+                'id': merchant['id'],
+                'field_name': 'Merchant',
+                'type': 'SELECT',
+                'options': merchant['options'],
+                'placeholder': merchant['placeholder'],
+                'category_ids': merchant['category_ids'],
+                'is_enabled': merchant['is_enabled'],
+                'is_custom': merchant['is_custom'],
+                'is_mandatory': merchant['is_mandatory'],
+                'code': merchant['code'],
+                'default_value': merchant['default_value'] if merchant['default_value'] else payload[0],
```

Comment:
There is no need to add a default value
```suggestion
                'default_value': merchant['default_value'] if merchant['default_value'] else None,
```

Reply:
This cannot be null if we are posting some options from our side. There will be a validation error.